src/Gui/Selection.cpp

```diff
@@ -507,7 +507,7 @@ bool SelectionSingleton::setPreselect(const char* pDocName, const char* pObjectN
                 msg = QCoreApplication::translate("SelectionFilter","Not allowed:");
             }
             msg.append(
-                        QObject::tr(" %1.%2.%3 ")
+                        QObject::tr("%1.%2.%3")

```

Comment:
How do I move prepend and append the whitespace to this string without including them in the string?

Reply:
Should be fixed in 0a43852

Reply:
" %1.%2.%3 " is a string that doesn't need to be translated at all. So replace the QObject::tr with QString::fromLatin1